cmd/influx_inspect/buildtsi/buildtsi.go

```diff
@@ -133,13 +184,18 @@ func (cmd *Command) processDatabase(dbName, dataDir, walDir string) error {
 }
 
 func (cmd *Command) processRetentionPolicy(sfile *tsdb.SeriesFile, dbName, rpName, dataDir, walDir string) error {
-	cmd.Logger.Info("rebuilding retention policy", logger.Database(dbName), logger.RetentionPolicy(rpName))
+	cmd.Logger.Info("Rebuilding retention policy", logger.Database(dbName), logger.RetentionPolicy(rpName))
 
 	fis, err := ioutil.ReadDir(dataDir)
 	if err != nil {
 		return err
 	}
 
+	var shards []struct {
+		ID   uint64
+		Path string
+	}
```

Comment:
Can you create a type for this struct so it doesn't have to be repeated below in the `append()`.